_data/navigation.js

```diff
@@ -98,102 +98,102 @@ module.exports = {
         contents: [
           {
             title: 'Introduction to Data Feeds',
-            url: '/docs/using-chainlink-reference-contracts/',
+            url: '/docs/data-feeds/',
           },
           {
-            title: 'Using Data Feeds',
-            url: '/docs/get-the-latest-price/',
+            title: 'Price Feeds',
+            url: '/docs/data-feeds/price-feeds/',
             children: [
               {
-                title: 'Data Feeds API Reference',
-                url: '/docs/price-feeds-api-reference/',
+                title: 'Historical Price Data',
+                url: '/docs/data-feeds/historical-data/',
+              },
+              {
+                title: 'Price Feeds API Reference',
+                url: '/docs/data-feeds/price-feeds/api-reference/',
+              },
+              {
+                title: 'Contract Addresses',
+                url: '/docs/data-feeds/price-feeds/addresses/',
+                children: [
```

Comment:
still to discuss what's the purpose of nested children since they are not supported by the current nav bar

Reply:
Removed from the nav and replaced in the Addresses page.